lib/mayaUsd/ufe/UsdPointInstanceOrientationModifier.cpp

```diff
@@ -40,6 +40,12 @@ MayaUsd::ufe::UsdPointInstanceOrientationModifier::Batches sBatches;
 namespace MAYAUSD_NS_DEF {
 namespace ufe {
 
+// Ensure that UsdPointInstanceOrientationModifier is properly setup.
+using base = UsdPointInstanceModifierBase<Ufe::Vector3d, PXR_NS::GfQuath>;
+MAYAUSD_VERIFY_CLASS_BASE(base, UsdPointInstanceOrientationModifier);
```

Comment:
Because the base class comes from Pixar using the macro "PXR_NS" - cannot use that in my macro. Thus the use of the "using" statement.